lib/font/font.toit

```diff
@@ -40,10 +43,20 @@ class Font:
     // The primitive takes a small array (not a list) of byte arrays, so we
     // convert to that.
     array := create_array_ glyph_byte_data
+    hash_code = calculate_hash_code_ array
     proxy_ = font_get_nonbuiltin_ resource_freeing_module_ array
     add_finalizer this:: this.finalize_
 
-  constructor.protected_ .proxy_:
+  static calculate_hash_code_ array/Array_ -> int:
```

Comment:
Maybe I'm missing something, but why does it make sense to have a hash code that is based on the font bits if == is based on identity?